src/propcache/_helpers_py.py

```diff
@@ -27,18 +45,27 @@ def __init__(self, wrapped: Callable[..., _T]) -> None:
         self.__doc__ = wrapped.__doc__
         self.name = wrapped.__name__
 
-    def __get__(self, inst: _TSelf[_T], owner: Optional[Type[Any]] = None) -> _T:
+    @overload
+    def __get__(  # pragma: no cover
+        self, inst: None, owner: Optional[Type[Any]] = None
```

Comment:
FTR, `Any` here is what makes MyPy think it's not fully covered.

Reply:
That’s unfortunate as Any is correct here (also similar to the standard lib functools cachedproperty type shead)

Reply:
Is it? The parameter isn't even used, so `object` would be type safe. Any just disables type checking.

Reply:
Yeah, I had the same thought. Just haven't had time to look into what should be put in here instead. I recently realized that such undercovered things also influence how coverage is computed in the calling projects, so it's quite important to improve the precision here.

Reply:
https://github.com/python/typeshed/blob/dbd0d3521745288a3b2e345d8683bb9539d28d60/stdlib/functools.pyi#L192

Here is what standard lib uses for `cached_property`

Reply:
If the implementation is type safe with anything, then it should be updated to object. I've had discussions in typeshed before, and the consensus is that typeshed should use object in such circumstances (otherwise, it triggers Any warnings in mypy if all the any warnings are enabled, anytime a user uses that function).

Reply:
attempt here https://github.com/aio-libs/propcache/pull/50

Reply:
@bdraco some of those types are historic due to limitations that existed in the past: https://github.com/python/mypy/issues/11878#issuecomment-1147925317.